tests/test_routes.py

```diff
@@ -497,6 +497,92 @@ def test_post_continue_application(self, flask_test_client, mocker):
 
         assert response.status_code == 302
         assert response.headers["Location"] == "/assess/application/app_123"
+
+    def test_qa_complete_flag_displayed(self, flask_test_client, mocker):
+        token = create_valid_token(test_lead_assessor_claims)
+        flask_test_client.set_cookie("localhost", "fsd_user_token", token)
+        mocker.patch(
```

Comment:
Im just conscious that the data we're using is diverging across PRs, some tests use test data in endpoint_json.json file and some a manual mock. I think the cleaner approach is probably using the .json file as we can also then see this when using flask run locally. Happy to discuss

Reply:
yeh definitely worth a chat! could be worth raising a separate BAU ticket to make these tests consistent - although I much prefer using a mock 😆

Reply:
yeah, Im happy o raise a BAU following our chat earlier. 
Test mocks and USE_LOCAL_DATA calls in endpoint_json.json for flask run, consuming the same data (local to test directory) is a good solution 👍